charts/kueue/templates/webhook/webhook.yaml

```diff
@@ -207,10 +207,31 @@ webhooks:
     apiVersions:
     - v1
     operations:
+    - CREATE
```

Comment:
This does not seem covered by the changes in `job_webhook_test.go`. Please extend that one.

Reply:
added test, which check JobMinParallelismAnnotation label